javascript-methods/main.js

```diff
@@ -0,0 +1,53 @@
+const x = 10;
+const y = 20;
+const z = 30;
+const maximumValue = Math.max(x, y, z);
+console.log('maximumValue:', maximumValue);
+
+const heroes = [
+  'ironMan',
+  'hulk',
+  'blackWidow',
+  'thor'
+];
+// const random = Math.floor(Math.random(heroes.length - 1));
+let randomNumber = Math.random();
+randomNumber *= heroes.length;
+const randomIndex = Math.floor(randomNumber);
+console.log('randomIndex:', randomIndex);
+const randomHero = heroes[randomIndex];
+console.log('randomHero:', randomHero);
+
+const library = [
+  { title: 'charolettesWeb', author: 'E.B.White' },
+  { title: 'harryPotter', author: 'J.K. Rowling' },
```

Comment:
For values, you don't need to use camelCase. Since they are strings, they can just use normal spacing / capitalization. So you can do _Charolettes Web_ & _Harry Potter_

Reply:
Thank you!